api/v1/errors/SqlError.js

```diff
@@ -0,0 +1,28 @@
+function checkError(err) {
```

Comment:
So I should return an `InvalidParameterError` instead of a `SQLError`? The `SQLError` also returns an specific error code such as er_dup_entry and er_no_referenced_row_2.